nomenclature/definitions/variable/economy/economy.yaml

```diff
@@ -4,6 +4,66 @@ Consumption:
    description: Total consumption of all goods, by all consumers in a region
    unit: billion US$2010/yr
 
+Consumption|Households|<product>|Imported:
+   description: Consumption of imported <this product> by households
+   unit: million euro/yr
+
+Consumption|Government|<product>|Domestic:
+   description: Consumption of domestic <this product> by households
+   unit: million euro/yr
+
+Consumption|Government|<product>|Imported:
+   description: Consumption of imported <this product> by government
+   unit: million euro/yr
+
+Consumption|Government|<product>|Domestic:
+   description: Consumption of domestic <this product> by government
+   unit: million euro/yr
+
+Consumption|GrossCapitalFormation|<product>|Imported:
+   description: Consumption of imported <this product> by gross capital formation
+   unit: million euro/yr
+
+Consumption|GrossCapitalFormation|<product>|Domestic:
+   description: Consumption of domestic <this product> by gross capital formation
+   unit: million euro/yr
+
+Consumption|Industry|<product>|Imported:
+   description: Consumption of imported <this product> by industries
+   unit: million euro/yr
+
+Consumption|Industry|<product>|Domestic:
+   description: Consumption of domestic <this product> by industries
+   unit: million euro/yr
+
+IndustryOutput|<industry>:
+   description: Total output on the <this industry> level
+   unit: million euro/yr
+
+ProductOutput|<product>:
+   description: Total output on the <this product> level
+   unit: million euro/yr
+
+Capital|<industry>:
+   description: Total capital costs spend by <this industry>
+   unit: million euro/yr
+
+Labor|<industry>:
+   description: Total wages spend by <this industry>
+   unit: million euro/yr
+
+ProductPriceIndex|<product>:
+   description: Price index on the <this product> level, relative to base year 2011
```

Comment:
Maybe only `Price index on <this product> level, relative to base year 2011`

Reply:
Got it!